tests/qml/tst_VPNCheckBoxRow.qml

```diff
@@ -59,6 +59,12 @@ Item {
             expected = false
             actual = checkboxrow.isChecked
             verify(expected === actual, `isChecked was ${actual} not ${expected}.`);
+
+            //Simulate clicking to the right of the label of the checkbox. The last parameter (2) represents the topMargin for the VPNInterLabel in VPNCheckBoxRow
+            mouseClick(checkboxrow, checkboxrow.width + 1, 2)
```

Comment:
suggestion (non-blocking): That’s definitely a smart alternative for emulating a click on the label. Just in case you are hitting the label indirectly here because the label is not accessible from the test: If you add an alias to the root item of `VPNCheckBoxRow` (e.g. `property alias checkboxLabel: label`) you should be able to access it via `checkboxrow.checkboxLabel`.

Reply:
yep, totally a valid option - I was just trying to get fancy with it to avoid adding an extra property since the label extends all the way to the end of the component